internal/controllers/topology/cluster/patches/engine.go

```diff
@@ -83,6 +83,14 @@ func (e *engine) Apply(ctx context.Context, blueprint *scope.ClusterBlueprint, d
 		clusterClassPatch := blueprint.ClusterClass.Spec.Patches[i]
 		ctx, log := log.WithValues("patch", clusterClassPatch.Name).Into(ctx)
 
+		patchName := clusterClassPatch.Name
```

Comment:
```suggestion
		definitionFrom := clusterClassPatch.Name
```
nit: I think we should call this `definitionFrom` or `definitionSource` (also in the func parameter)